src/Collector.cpp

```diff
@@ -58,16 +62,17 @@ std::vector<CollectorPtr> collectorsCreate(ListOf<List> specs,
 
 // Implementations ------------------------------------------------------------
 
-void CollectorCharacter::setValue(int i, const Token& t) {
-  switch(t.type()) {
+void CollectorCharacter::setValue(int i, const Token &t) {
```

Comment:
I think relevant options are `DerivePointerAlignment: true` (which will use the most common alignment of `&` and `*`, or `PointerAlignment: Left` which would always align pointers / references with the left.